src/MizarWidgets/include/MizarWidgets/SamplingWithFrameTrackInputWidget.h

```diff
@@ -86,6 +93,30 @@ class SamplingWithFrameTrackInputWidgetTmpl : public SamplingWithFrameTrackInput
       list->addItem(item.release());
     }
   }
+
+  void InitFrameTrackList(const PairedData& data) {
+    const absl::flat_hash_map<uint64_t, orbit_client_data::ScopeInfo> scope_id_to_info =
+        data.GetFrameTracks();
+    std::vector<std::pair<uint64_t, orbit_client_data::ScopeInfo>> scope_id_to_info_sorted(
+        std::begin(scope_id_to_info), std::end(scope_id_to_info));
+    std::sort(std::begin(scope_id_to_info_sorted), std::end(scope_id_to_info_sorted),
+              [](const auto& a, const auto& b) { return a.second.GetName() < b.second.GetName(); });
+
+    for (size_t i = 0; i < scope_id_to_info_sorted.size(); ++i) {
+      const auto& [scope_id, scope_info] = scope_id_to_info_sorted[i];
+      GetFrameTrackList()->insertItem(i, MakeFrameTrackString(scope_info));
+      GetFrameTrackList()->setItemData(i, QVariant::fromValue(scope_id), kScopeIdRole);
+    }
+    OnFrameTrackSelectionChanged(0);
+  }
+
+  [[nodiscard]] QString MakeFrameTrackString(const orbit_client_data::ScopeInfo& scope_info) const {
```

Comment:
Is there a reason for this being a member function? I think it could be a free or static function since it's not accessing `this`, right?

Reply:
No reason at all. I'll make it static.